dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ClassDumper.java

```diff
@@ -184,10 +187,9 @@ static SymbolTable scanClassSymbolTable(JavaClass javaClass) {
     return symbolTableBuilder.build();
   }
 
-  static MethodSymbolReference constantToMethodReference(ConstantMethodref constantMethodref,
-      ConstantPool constantPool, String sourceClassName) {
-    String classNameInMethodReference = constantMethodref.getClass(constantPool);
-    int nameAndTypeIndex = constantMethodref.getNameAndTypeIndex();
+  static ConstantNameAndType constantNameAndTypeFromConstantCP(
```

Comment:
You could just call this "constantNameAndType", since we know it's a `ConstantCP` (whatever that is) from the parameters.

Reply:
Renamed to `constantNameAndType`.